Source/Wit/Public/Wit/Configuration/WitAppConfiguration.h

```diff
@@ -10,35 +10,63 @@
 #include "WitAppConfiguration.generated.h"
 
 /**
- * Wit general application configuration
+ * Wit advanced application configuration
  */
 USTRUCT(BlueprintType)
-struct WIT_API FWitAppConfiguration
+struct WIT_API FWitAppAdvancedConfiguration
 {
 	GENERATED_BODY()
 
-	/**
-	 * The client access token to use when calling Wit. This can be obtained from your App setup on Wit.ai
-	 */
-	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Access Token")
-	FString ClientAccessToken{};
-
 	/**
 	 * Specifies the base URL to use when making requests to Wit.ai. If left empty this will use the default base URL
 	 */
-	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Endpoint")
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Request")
```

Comment:
Lets call this "Request Overrides" to be clear that they don't actually need to change any of the values.

Reply:
I feel this is ok.  we will set the default value here for each release.   Users will be able to see the version here, it is up to them if they want to change it.

Reply:
Changed this